qbistListeners.js

```
@@ -26,40 +33,40 @@ function saveState() {
     } else {
       alert("Invalid pattern state")
     }
   } catch (e) {
+    console.error("Error loading state:", e)
     alert("Error loading pattern state")
   }
 }
 // when a preview is clicked, use its formula as the new main pattern
 document.querySelectorAll(".preview").forEach((canvas) => {
   canvas.addEventListener("click", () => {
     const index = parseInt(canvas.id.replace("preview", ""))
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)